mhs/common/mhs_common/workflow/__init__.py

```diff
@@ -1,21 +1,26 @@
 """Modules related to management of the workflow for supported messaging patterns."""
 from typing import Dict
 
+from comms import transmission_adaptor
+
+from mhs_common.state import persistence_adaptor
 from mhs_common.workflow.asynchronous_express import AsynchronousExpressWorkflow
 from mhs_common.workflow.asynchronous_reliable import AsynchronousReliableWorkflow
 from mhs_common.workflow.common import CommonWorkflow
 from mhs_common.workflow.intermediary_reliable import IntermediaryReliableWorkflow
 from mhs_common.workflow.synchronous import SynchronousWorkflow
 
 
-def get_workflow_map() -> Dict[str, CommonWorkflow]:
+def get_workflow_map(persistence_store: persistence_adaptor.PersistenceAdaptor,
+                     transmission: transmission_adaptor.TransmissionAdaptor,
```

Comment:
Should this default to `None`? Otherwise the inbound service will have to pass a `TransmissionAdaptor`. I don't think inbound needs the persistence adaptor either since that's passed to the actual handler to determine workflow then the work flow just uses the work description.